cmd/main.go

```diff
@@ -201,6 +202,26 @@ func maxDuration(dur time.Duration, durations ...time.Duration) time.Duration {
 	return ret
 }
 
+func WrapServeError() func(rw http.ResponseWriter, r *http.Request, err error) {
```

Comment:
Can you put this with the rest of the middleware in [`pkg/app/middleware.go`](https://github.com/openshift/assisted-service/blob/4476f900d22e1019c04c4ef4201cc45377a9eadc/pkg/app/middleware.go)?

Reply:
done